pkg/kpack/build_client.go

```diff
@@ -0,0 +1,88 @@
+package kpack
+
+import (
+	"github.com/pivotal/kpack/pkg/apis/build"
+	v1alpha2 "github.com/pivotal/kpack/pkg/apis/build/v1alpha2"
+	"github.com/pivotal/kpack/pkg/client/clientset/versioned/scheme"
+	kpackV1alpha2 "github.com/pivotal/kpack/pkg/client/clientset/versioned/typed/build/v1alpha2"
+	rest "k8s.io/client-go/rest"
+)
+
+const (
+	kpackGroup                = build.GroupName
+	kpackVersionV1alpha2      = "v1alpha2"
+	kpackGroupVersionV1alpha2 = kpackGroup + "/" + kpackVersionV1alpha2
+)
+
+// KpackV1alpha2Client is used to interact with features provided by the kpack.io group.
+type KpackV1alpha2Client struct {
```

Comment:
let's call this something about compatibility or conversion like `KpackV1alpha1CompatClient`

Reply:
also I think this file can be named `client.go`